build.gradle

```diff
@@ -343,17 +343,29 @@ task cmakeJniLib(type:Exec) {
         args.add("-DBLAS_LIBRARIES=$rootDir\\src\\main\\resources\\windowsDependencies\\libopenblas.dll")
         args.add("-DLAPACK_LIBRARIES=$rootDir\\src\\main\\resources\\windowsDependencies\\libopenblas.dll")
     }
+    //if (System.getenv("LIBNAME") == "nmslib_linux_x64") {
```

Comment:
can you cleanup all these commented lines

Reply:
Updated.